doc/source/programs/gdal_raster_calc.rst

```diff
@@ -0,0 +1,104 @@
+.. _gdal_raster_calc_subcommand:
+
+================================================================================
+"gdal raster calc" sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Perform raster algebra
+
+.. Index:: gdal raster calc
+
+Synopsis
+--------
+
+.. program-output:: gdal raster calc --help
+    :ellipsis: -5
+
+
+Description
+-----------
+
+:program:`gdal raster calc` performs pixel-wise calculations on one or more input GDAL datasets. Calculations
+can be performed eagerly, writing results to a conventional raster format,
+or lazily, written as a set of derived bands in a :ref:`VRT (Virtual Dataset) <raster.vrt>`.
+
+The list of input GDAL datasets can be specified at the end
+of the command line or put in a text file (one input per line) for very long lists.
+If more than one input dataset is used, it should be prefixed with a name by which it
+will be referenced in the calculation, e.g. ``A=my_data.tif``. (If a single dataset is
+used, it will be referred to with the variable ``X`` in formulas.)
+
+The inputs should have the same spatial reference system and should cover the same spatial extent but are not required to have the same
+spatial resolution. The spatial extent check can be disabled with :option:`--no-check-extent`,
+in which case the inputs must have the same dimensions. The spatial reference system check can be
+disabled with :option:`--no-check-srs`.
+
+The following options are available:
+
+.. include:: gdal_options/of_raster_create_copy.rst
+
+.. include:: gdal_options/co.rst
+
+.. include:: gdal_options/overwrite.rst
+
+.. option:: -i [<name>=]<input>
+
+    Select an input dataset to be processed. If more than one input dataset is provided,
+    each dataset must be prefixed with a name to which it will will be referenced in :option:`--calc`.
+
+.. option:: --calc
+
+    An expression to be evaluated using the `muparser <https://beltoforion.de/en/muparser>`__ math parser library.
+    The expression may refer to individual bands of each input (e.g., ``X[1] + 3``) or it may be applied to all bands
+    of an input (``X + 3``). If the expression contains a reference to all bands of multiple inputs, those inputs
+    must either have the same the number of bands, or a single band.
+
+    For example, if inputs ``A`` and ``B`` each have three bands, and input ``C`` has a single band, then the argument
+    ``--calc "A + B + C"`` is would be equivalent to ``--calc "A[1] + B[1] + C[1]" --calc "A[2] + B[2] + C[2]" --calc "A[3] + B[3] + C[3]"``.
+
```

Comment:
I don't undestand the example. If C has one band, is is really possible to use `C[2]` and `C[3]`? Why?
"is would be equivalent to" should probably be "would be equivalent".

Reply:
A sloppy mistake, fixed in 648dfd983511ac6b4e4b27b4cd9e0722bc3df15e